docs/pages/run/validator-management/vc-configuration.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Configuration
+title: Stake with a Validator Client
```

Comment:
hmm, again redundant to mentioned "Validator Client" here, and what else are you gonna do with a validator client besides staking?